packages/neo-one-website/package.json

```diff
@@ -18,6 +19,8 @@
     "@types/react-redux": "^6.0.9",
     "constate": "^0.8.2",
     "fs-extra": "^7.0.0",
+    "gray-matter": "^4.0.1",
+    "highlight.js": "^9.12.0",
```

Comment:
I added prism for highlighting, can you rebase and revert the highlight.js changes.